<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeDialog</class>
 <widget class="QDialog" name="TreeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>468</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Tree parameters</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget, QMenuBar::item, QHeaderView::section {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #c5d8ef, stop: 1 #89a5c3);
}

QLabel, QSlider {
	background-color: transparent;
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Number of root routers:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="routersNum0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="rootLevel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Root layer</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Router bandwidth:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="routerBandwidth0">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>93</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label1">
          <property name="text">
           <string>Number of outgoing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label2">
          <property name="text">
           <string>channels per router:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="routerChilds0">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>93</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Channels bandwidth:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="channelsBandwidth0">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="numRouters1Lab">
       <property name="text">
        <string>Number of 1st layer routers:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="routersNum1">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="level1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>1st layer</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Router bandwidth:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="routerBandwidth1">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>28</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Number of outgoing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>channels per router:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="routerChilds1">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>28</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Channels bandwidth:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="channelsBandwidth1">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="level2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>2nd layer</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Router bandwidth:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="routerBandwidth2">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>28</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Channels bandwidth:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="channelsBandwidth2">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>22</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Number of computers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>per router:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="computersNum"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>23</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Number of storages</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>per router:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="storagesNum">
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Computer performance:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Number of storage types:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Storage capacity:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QSpinBox" name="performance">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numTypes">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="capacity">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>118</width>
         <height>38</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>OK</sender>
   <signal>clicked()</signal>
   <receiver>TreeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>TreeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddVersion()</slot>
  <slot>RemoveVersion()</slot>
 </slots>
</ui>
